std_msgs/Header header
std_msgs/Int32 en_arm
float64[3] C1
float64[3] C2
float64[3] C3
float64[3] C4
float64[3] umf
float64[3] vsf
geometry_msgs/PoseStamped master_pose
geometry_msgs/PoseStamped slave_pose
